### ISSUE TITLE
Add Discord icon on Toolpad landing page

### ISSUE THREAD

**Issue by prakhargupta1** [mui/mui-toolpad#2235]
### Duplicates

- [X] I have searched the existing issues

### Latest version

- [X] I have tested the latest version

### Summary 💡

Toolpad users/explorers should be able to quickly find the Discord icon if they want to join the community space.
Related to: https://github.com/mui/material-ui/issues/37447

Discord server link: https://discord.gg/hHqtMP9Ckc

# Benchmarks 🌈

## Appsmith
At the bottom of their homepage and footer.
They further link their Discord in many of their docs pages. Eg: https://docs.appsmith.com/connect-data/reference/querying-mysql#troubleshooting

## Windmill
1. At the top nav bar and footer
<img width="1248" alt="Screenshot 2023-06-28 at 5 49 03 PM" src="https://github.com/mui/mui-toolpad/assets/92228082/ad1766c2-8d35-4b5a-8fe3-e8cc7e183e47">
2. Top nav bar of docs: https://www.windmill.dev/docs/intro 

## Refine.dev
At 2 places in top nav bar and in the footer
<img width="1407" alt="Screenshot 2023-06-28 at 5 47 37 PM" src="https://github.com/mui/mui-toolpad/assets/92228082/17bc50b8-33c0-4f28-8698-388ed11a4761">
Also, in the Getting started page in docs: https://refine.dev/docs/#community

## React admin
In the top nav bar of homepage and docs.
<img width="1343" alt="Screenshot 2023-06-28 at 5 50 15 PM" src="https://github.com/mui/mui-toolpad/assets/92228082/da50868d-95c2-4eeb-9ff5-63ac958c3d60">

## Interval
An icon in the footer and docs support page: https://interval.com/docs/support 
They don't have it on the header nav bar. It could be one of the reasons for having the lowest number of discord members among all other listed.

**Comment by Janpot** [comment]
@prakhargupta1 Do you mean to include it in the footer, like

<img width="765" alt="Screenshot 2023-06-30 at 12 49 26" src="https://github.com/mui/mui-toolpad/assets/2109932/464f4296-e8cd-4443-a5d5-b1384a353f21">

**Comment by prakhargupta1** [comment]
> @prakhargupta1 Do you mean to include it in the footer, like
> 
> <img alt="Screenshot 2023-06-30 at 12 49 26" width="765" src="https://user-images.githubusercontent.com/2109932/250069871-464f4296-e8cd-4443-a5d5-b1384a353f21.png">

Yes, definitely here. 
@richbustos This is an issue that I created for Toolpad but can cover the entire mui.com scope.

**Comment by Janpot** [comment]
Yes, this will be handled in the core repo after which we'll pull the changes on our weekly update.

**Comment by richbustos** [comment]
@prakhargupta1 Thanks for doing this! I can add the discord link on the core footer once I finish with announcement plans for the discord. Thank you!
